### PR TITLE
[skip ci] Update readme to use hud.pytorch.org

### PR DESCRIPTION
**Pull request by driazati**
Differential Revision: [D28632504](https://our.internmc.facebook.com/intern/diff/28632504/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ You can reuse your favorite Python packages such as NumPy, SciPy, and Cython to
...
 | Linux (aarch64) CPU | [![Build Status](http://openlabtesting.org:15000/badge?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py36)](https://status.openlabtesting.org/builds/builds?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py36) | [![Build Status](http://openlabtesting.org:15000/badge?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py37)](https://status.openlabtesting.org/builds/builds?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py37) | [![Build Status](http://openlabtesting.org:15000/badge?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py38)](https://status.openlabtesting.org/builds/builds?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py38) |
 
-See also the [ci.pytorch.org HUD](https://ezyang.github.io/pytorch-ci-hud/build/pytorch-master).
+See also the [ci.pytorch.org HUD](https://hud.pytorch.org/build2/pytorch-master).
```
Comment (janeyx99) [inline README.md:51]: Clicking on the link leads to a long time of redirects for me, looking like it expands to: 
`https://hud.pytorch.org/build2/?p=/&q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~...and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/~and~q=p=/pytorch-master`

Comment (janeyx99) [reply README.md:51]: Oh yea, did you mean to have
```suggestion
See also the [ci.pytorch.org HUD](https://ezyang.github.io/pytorch-ci-hud/build2/pytorch-master).
```

Comment (janeyx99) [reply README.md:51]: But it seems a first-few times thing, I retried it again and it doesn't have the same problem...

Comment (driazati) [reply README.md:51]: Should be handled in https://github.com/ezyang/pytorch-ci-hud/pull/43

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**